.github/workflows/dependabot_update_rbis_run.yml

```
@@ -0,0 +1,9 @@
+name: Update RBIs
```

```suggestion
name: Update gem RBIs on Dependabot PRs
```
I'd like to have the reusable workflow in tapioca and name this more specifically.